Enable `history undo` tests

kontura opened the pull request:
Requires: https://github.com/rpm-software-management/dnf5/pull/1452 and https://github.com/rpm-software-management/dnf5/pull/1454

pep8speaks commented:
Hello @kontura! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`dnf-behave-tests/dnf/steps/lib/dnf.py`](https://github.com/rpm-software-management/ci-dnf-stack/blob/4a84cd7b9975c6ff2b3cb1213cd091b36ac636b4/dnf-behave-tests/dnf/steps/lib/dnf.py):

> [Line 330:41](https://github.com/rpm-software-management/ci-dnf-stack/blob/4a84cd7b9975c6ff2b3cb1213cd091b36ac636b4/dnf-behave-tests/dnf/steps/lib/dnf.py#L330): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 330:85](https://github.com/rpm-software-management/ci-dnf-stack/blob/4a84cd7b9975c6ff2b3cb1213cd091b36ac636b4/dnf-behave-tests/dnf/steps/lib/dnf.py#L330): [E226](https://duckduckgo.com/?q=pep8%20E226) missing whitespace around arithmetic operator

pkratoch reviewed (approved):
LGTM, waiting for the dnf5 patch to merge.

kontura commented:
Updated the tests to match new messages.

pkratoch commented on dnf-behave-tests/dnf/module/install-package-ursine.feature:
```diff
@@ -32,7 +32,7 @@ Scenario: I can't reinstall installed non-modular content which is masked by act
    When I execute dnf with args "reinstall wget"
    Then the exit code is 1
-    And stderr contains "Packages for argument 'wget' installed and available, but in a different version => cannot reinstall"
+    And stderr contains "Installed packages for argument 'wget' are not available in repositories, available versions: wget-1.18.5-5.module.x86_64, cannot reinstall."
```
I think it's a bit confusing that the message says that the packages are not available. It would be better if it said "are not available in repositories in the same version".

kontura replied:
That is better, updated, thanks!

kontura commented:
/packit test https://github.com/rpm-software-management/dnf5/pull/1452

kontura commented:
I have rebased the PR and also some of the tests needed an update.
Most notably there was a lot of failing `package reasons are` steps, I have added their updates to the `Enable step package reasons are for dnf5` commit.

kontura commented:
> @kontura thanks for the update. I suppose you want to merge the last commit [merge into enable undo](https://github.com/rpm-software-management/ci-dnf-stack/pull/1497/commits/15fd7c3a9fc2975bbb52b255f6ead1009ee8687b) into [Enable history-undo.feature](https://github.com/rpm-software-management/ci-dnf-stack/pull/1497/commits/2a800fdad625d47e2a0f742c71c4dbf31f6c310d) before I merge this?

oh right, yes, thank you. :slightly_smiling_face:
